spec/support/mockers/mocker.rb

```diff
@@ -0,0 +1,70 @@
+require 'rspec/mocks/standalone'
+
+class Mocker < RSpec::Mocks::InstanceVerifyingDouble
+  include RSpec::Mocks
+
+  FIND_METHODS = [:find, :find_by_id]
+
+  def initialize(traits: [], **attrs)
+    # from RSpec::Mocks::ExampleMethods
+    # combination of #declare_verifying_double and #declare_double
+    ref = ObjectReference.for(self.class.klass_name)
+    RSpec::Mocks.configuration.verifying_double_callbacks.each do |block|
+      block.call(ref)
+    end
+    attrs ||= {}
+    super(ref, attrs)
+    self.as_null_object
```

Comment:
Oh I see, so it just responds `nil` to any message (presumably using `method_missing` or something along those lines), allowing you to make `have_received` assertions without explicitly mocking those methods. Very cool!